core/app/models/spree/credit_card.rb

```diff
@@ -91,7 +91,6 @@ def cc_type=(type)
 
     # Sets the last digits field based on the assigned credit card number.
     def set_last_digits
-      number.to_s.gsub!(/\s/, '')
```

Comment:
Does this mean that we no longer automatically strip whitespace from credit card numbers at the model level? Is that worth a changelog entry?